### PR TITLE
Vector setup

### PR DESCRIPTION
**Pull request by AbbyGi**
Replaced `setPvDesc` calls for vector program device in `zebraDaq` function. Still needs to be tested.

### CODE REVIEW

**Review by JunAishima — CHANGES_REQUESTED** [review]
to me, everything looks good, except for the import.

please fix up the bluesky name in the import. I am happy otherwise.

**Comment by JunAishima** [comment]
I am also concerned about this being rebased on my prepare_zebra branch (which has not been approved and merged yet). please consider this when reviewing.

while unconventional, doing this has the advantage of add onto my existing scans.py module.

**Review by mrakitin — APPROVED** [review]
I don't see major issues with that update.

As a side note, we actually need to enable linting (`flake8`+`black` auto-formatting).

File: scans.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#objects available should be zebra, vector, eiger
+import bluesky.plan_stubs as bps
+from start_bs import zebra, vector_program
+
+def zebra_daq_prep():
+       RESET_GROUP = 'reset'
+       yield from bps.abs_set(zebra.reset, 1, group=RESET_GROUP)
+       yield from bps.wait(RESET_GROUP)
```
[the remark this answers is not in the document]
Comment (JunAishima) [reply scans.py:8]: OK, we have chatted in the weekly meeting and it is clear that I must have misunderstood that I would need to do abs_set and wait before sleep. it is now obvious to me that I didn't need to do this. Will put in a commit to fix this

[the remark this answers is not in the document]
Comment (JunAishima) [reply scans.py]: fixed with https://github.com/NSLS-II/lsdc/pull/54/commits/ea1fa36b3c349c42944c0b296a52678388af8601

File: scans.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                      zebra.pc.pulse.start, 0,
+                      zebra.pc.pulse.width, pulse_width,
+                      zebra.pc.pulse.step, pulse_step,
+                      zebra.pc.pulse.delay, exposure_period_per_image / 2 * 1000,
+                      zebra.pc.pulse.max, num_images)
+
+def setup_zebra_vector_scan_for_raster(angle_start, image_width, exposure_time_per_image, exposure_period_per_image, detector_dead_time, num_images, scan_encoder=3):
+    VECTOR_SCAN_GROUP = 'vector_scan'
+    yield from bps.abs_set(zebra.pc.encoder, scan_encoder, group=VECTOR_SCAN_GROUP)
+    yield from bps.wait(VECTOR_SCAN_GROUP)
```
Comment (mrakitin) [inline scans.py:29]: This place should also be resolved to use `bps.mv(...)` instead.

**Review by JunAishima — APPROVED** [review]

**Review by mrakitin — APPROVED** [review]

**Pull request merged**